janitor/functions.py

```diff
@@ -5514,38 +5514,34 @@ def groupby_topk(
 
 
 @pf.register_dataframe_method
-def complete(
-    df: pd.DataFrame,
-    *columns,
-    by: Optional[Union[list, str]] = None,
-) -> pd.DataFrame:
-    """
-    This function turns implicit missing values into explicit missing values.
 
-    It is modeled after tidyr's `complete` function, and is a wrapper around
-    `expand_grid`, `pd.DataFrame.reindex`, `pd.DataFrame.join`
-    and `pd.DataFrame.fillna`.
+    It is modeled after tidyr's ``complete`` function, and is a wrapper around
+    ``expand_grid``, ``pd.DataFrame.reindex``, ``pd.DataFrame.join``
+    and ``pd.DataFrame.fillna``.
```

Comment:
@samukweku I think we can use single back-ticks here. A simple find-and-replace should do the trick.

Reply:
FYI I'm happy to handle this :smile:.